README.md

```diff
@@ -3,11 +3,12 @@
 ## Description
 This plugin manages how players trade one item for another. The concept behind it is very simple, you just place a chest and a sign on top, then write all the information on the sign.
 
```

Comment:
Does this need to be re written now?

Reply:
Why?

Reply:
You right, its was just the write info on the sign part, but that can still be done. so this is good.